anni/src/subcommands/repo.rs

```diff
@@ -238,6 +246,64 @@ fn repo_get_vgmdb(options: &RepoGetVGMdb, manager: &RepositoryManager, get: &Rep
     Ok(())
 }
 
+#[derive(Parser, Debug, Clone)]
+pub struct RepoGetCue {
+    #[clap(short = 'p', long)]
+    path: String,
+    /// if metadata is not enough, search vgmdb with keyword
+    #[clap(short = 'k', long)]
+    keyword: Option<String>,
+}
+
+#[handler(RepoGetCue)]
+fn repo_get_cue(
+    options: &RepoGetCue,
+    manager: &RepositoryManager,
+    get: &RepoGetAction,
+) -> anyhow::Result<()> {
+    let path = &options.path;
+
+    let cue = Cuna::open(path)?;
+    let mut album = match (cue.catalog(), options.keyword.as_ref()) {
+        // if catalog is found, fetch metadata from vgmdb
+        (Some(catalog), _) => search_album(&catalog.to_string()).await?,
+        // otherwise try to search with keyword
+        (None, Some(keyword)) => {
+            warn!(
```

Comment:
Use `info!` instead of `warn!`?

Reply:
Operation below is likely to fail and produce wrong results, so I think `warn!` is more appropriate.